ivadomed/prepare_dataset.py

```diff
@@ -0,0 +1,215 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import PIL
+import skimage
+import os
+import sys
+from utils import reorient_image
+
+# normalize Image
+def normalize(arr):
+    ma = arr.max()
+    mi = arr.min()
+    return ((arr - mi) / (ma - mi))
+
+
+# Useful function to generate a Gaussian Function on given coordinates. Used to generate groudtruth.
+def label2maskmap_gt(data, shape, c_dx=0, c_dy=0, radius=10, normalize=False):
```

Comment:
maybe move to `ivadomed/utils.py`?

Reply:
I think it can be done in subsequent PR with ivadomed/maths (as this is a convolution)

Reply:
> I think it can be done in subsequent PR with ivadomed/maths (as this is a convolution)

agreed, can you pls open an issue and link it here?

Reply:
issue open [here](https://github.com/ivadomed/ivadomed/issues/319) about the creation of ivadomed/maths

Reply:
@lrouhier it would be helpful if you could, in the issue, already list some of the functions already identified in this thread

Reply:
I am doing it right now.